.github/dependabot.yml

```
@@ -1,15 +1,24 @@
-      interval: "weekly"
-    cooldown:
-      default-days: 7
-  - package-ecosystem: "npm"
-    directory: "/"
-    schedule:
-      interval: "weekly"
-    cooldown:
-      default-days: 7
-      
\ No newline at end of file
+version: 2
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)